Converter/app/git.gradle

```diff
@@ -0,0 +1,41 @@
+def getPatch() {
+    def diffOutput = new ByteArrayOutputStream()
+    exec {
+        commandLine 'cmd', 'git', 'describe'
+        standardOutput = diffOutput
+    }
+    def res = diffOutput.toString()
+    if (res.contains('-')) {
```

Comment:
In which case we will see this condition return `true`?

Reply:
I made changes to git. gradle. It turned out that the 'cmd' command was unnecessary and returned the wrong thing. Now it returns true if tag exists